data/meetups.yml

```diff
@@ -3,47 +3,115 @@ template:
   map: ""
   address: ""
   venue: ""
+  display_date: ""
   date: 2018-10-03T19:00:00+08:00
   dates: "20181003T190000/20181003T210000"
   link: ""
+  group: ""
 
 mindvalley: &mindvalley
+  venue: "Mindvalley Hall of Awesomeness"
   map: "https://goo.gl/maps/n5rZ7Nfv3qs"
   address: "Level 29 Tower A Menara UOA Bangsar Kuala Lumpur, Malaysia"
-  venue: "Mindvalley Hall of Awesomeness"
 
 suria_labs: &suria_labs
+  venue: "Suria Labs"
   map: "https://goo.gl/maps/ik9uWtxxKs52"
   address: "B-12-6, Level 12, Tower B, Vertical Office Suites, Jalan Kerinchi, Bangsar South Kuala Lumpur, Malaysia"
-  venue: "Suria Labs"
+
+sunway_university: &sunway_university
+  venue: "Sunway University"
+  map: "https://goo.gl/maps/Z2sVv3FV2C92"
+  address: "5, Jalan Universiti, Bandar Sunway, 47500 Subang Jaya, Selangor"
+
+kdu_university_glenmarie: &kdu_university_glenmarie
+  venue: "KDU University Glenmarie"
+  map: "https://goo.gl/maps/zn7NQ7ec8qT2"
+  address: "Jalan Kontraktor U1/14, Seksyen U1, Glenpark U1, 40150 Shah Alam, Selangor"
+
+nu_sentral: &nu_sentral
+  venue: "NU Sentral"
+  map: "https://goo.gl/maps/gPxyS42heJv"
+  address: "201, Jalan Tun Sambanthan, Brickfields, 50470 Kuala Lumpur, Wilayah Persekutuan Kuala Lumpur"
+
+tbc: &tbc
+  venue: "Venue to be confirmed"
 
 list:
 - name: "RubyConf MY 2018"
   map: "https://goo.gl/maps/eMeuapNw36L2"
   address: "Menara Ta One, KLCC 22, Jalan P. Ramlee, Kuala Lumpur"
   venue: "One Auditorium"
+  display_date: Thu 25 - 26 October 2018, 8:00am - 5:00pm
```

Comment:
I was actually thinking of using javascript to read the original date and format it for human readability to reduce duplication in the source files.

Do you want to try that before merging?

Reply:
Here's what I'm thinking of:
```
date: 2018-10-25T08:00:00+08:00
until: 2018-10-26T17:00:00+08:00
```

then we do the `dates` key to be parsed by js and autogenerated.

Reply:
Okay, I think we'll merge it first and I can just do a simple implementation for this one before the data file becomes too large :P

Reply:
Sure! I'm all for using javascript, it took a while to format the dates. Easy to make mistakes also. I'd need to learn how to do that though.

Reply:
Also feel free to remove the events that you've mentioned earlier.